modules/olm-overriding-operator-pod-affinity.adoc

```diff
@@ -0,0 +1,254 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-scheduler-node-affinity.adoc
+// * nodes/scheduling/nodes-scheduler-pod-affinity.adoc
+// * operators/admin/olm-adding-operators-to-cluster.adoc
+
+ifeval::["{context}" == "nodes-scheduler-pod-affinity"]
+:pod:
+endif::[]
+ifeval::["{context}" == "nodes-scheduler-node-affinity"]
+:node:
+endif::[]
+ifeval::["{context}" == "olm-adding-operators-to-a-cluster"]
+:olm:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="olm-overriding-operator-pod-affinity_{context}"]
+
+ifdef::olm[]
+= Controlling where an Operator is installed
+endif::olm[]
+
+ifdef::pod[]
+= Using pod anti-affinity to control where an Operator is installed
+endif::pod[]
+
+ifdef::node[]
+= Using node affinity to control where an Operator is installed
+endif::node[]
+
+By default, when you install an Operator, {product-title} installs the Operator pod to one of your worker nodes randomly. However, there might be situations where you want that pod scheduled on a specific node or set of nodes.
+
+For example, some Operators require being scheduled on a node with a particular architecture, such as linux/amd64, linux/arm64, or Windows.
+
+Or you might want Operators that work closely together to be scheduled on the same host, or on hosts located on the same rack. Alternatively, you might want your Operators dispersed throughout the infrastructure to avoid downtime due to network or hardware issues.
+
+You can control where an Operator pod is installed by adding node affinity or pod anti-affinity constraints to the Operator's `Subscription` object. Node affinity is a set of rules used by the scheduler to determine where a pod can be placed. Pod anti-affinity allows you to restrict which nodes your pod can be scheduled on.
+
+ifdef::olm[]
+The following examples show how to use node affinity or pod anti-affinity to install an instance of the Custom Metrics Autoscaler Operator to a specific node in the cluster.
+endif::olm[]
+ifdef::node[]
+The following examples show how to use node affinity to install an instance of the Custom Metrics Autoscaler Operator to a specific node in the cluster.
```

Comment:
```suggestion
The following examples show how to use node affinity or pod anti-affinity to install an instance of the Custom Metrics Autoscaler Operator to a specific node in the cluster:
endif::olm[]
ifdef::node[]
The following examples show how to use node affinity to install an instance of the Custom Metrics Autoscaler Operator to a specific node in the cluster:
```
I think these lead-in sentences needs a colon at the end of the lead-in sentence [based on the ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording).